### PR TITLE
Updating Metabase to 0.31.2

### PR DESCRIPTION
**Pull request by jevy**
@pmint93

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/metabase/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 description: The easy, open source way for everyone in your company to ask questions
   and learn from data.
 name: metabase
-version: 0.4.4
-appVersion: v0.30.1
+version: 0.4.5
+appVersion: v0.31.2
```
Comment (davidkarlsen) [inline stable/metabase/Chart.yaml:6]: please update readme too

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by jevy** [comment]
@davidkarlsen Should be good.

**Comment by materemias** [comment]
I think @davidkarlsen 's requests were fulfilled. is there anything left to do?

**Pull request closed without merging**